### PR TITLE
style: improve the "Create New Chart" screen's usability

### LINKED ISSUE #11299: Usability - button color and styling
Screen Shot 2020-10-15 at 9 04 53 PM

Let's change the data viz select button's color - from grey to CTA colors like blue; please consider changing the button styling below as well. There are a few others in the explore control panel. We should get to them all gradually. @rusackas 🙏

### PR DESCRIPTION
**Pull request by mistercrunch**
### SUMMARY
more details in  https://github.com/apache/incubator-superset/issues/11299

closes https://github.com/apache/incubator-superset/issues/11299

#### after
Screen Shot 2020-10-21 at 9 31 17 PM

#### before 

imga

### TEST PLAN
🙏

### CODE REVIEW

**Review by nytai — APPROVED** [review]

**Comment by nytai** [comment]
Does this actually address #11299? That issue seems to be about the label button for the viz type

File: superset-frontend/src/addSlice/AddSliceContainer.tsx [hunk 1]
```diff
@@ -103,7 +103,7 @@ export default class AddSliceContainer extends React.PureComponent<
                   name="select-datasource"
                   onChange={this.changeDatasource}
                   options={this.props.datasources}
-                  placeholder={t('Choose a datasource')}
+                  placeholder={t('Choose a dataset')}
```
Comment (mistercrunch) [inline superset-frontend/src/addSlice/AddSliceContainer.tsx:106]: bycatch

File: superset-frontend/src/components/Button/index.tsx [hunk 2]
```diff
@@ -221,7 +221,7 @@ const SupersetButton = styled(BootstrapButton)`
     &[disabled],
     &[disabled]:hover {
       background-color: ${({ theme }) => theme.colors.grayscale.light2};
-      color: ${({ theme }) => theme.colors.grayscale.light1};
+      color: ${({ theme }) => theme.colors.grayscale.base};
```
Comment (mistercrunch) [inline superset-frontend/src/components/Button/index.tsx:224]: makes disabled button a bit more readable/accessible

File: superset-frontend/src/addSlice/AddSliceContainer.tsx [hunk 3]
```diff
@@ -134,6 +134,7 @@ export default class AddSliceContainer extends React.PureComponent<
                 name="select-vis-type"
                 onChange={this.changeVisType}
                 value={this.state.visType}
+                labelBsStyle="primary"
```
Comment (mistercrunch) [inline superset-frontend/src/addSlice/AddSliceContainer.tsx:137]: more of a CTA on choosing the viz type

**Comment by mistercrunch** [comment]
@nytai sorry about the confusion, I ended up fixing a few things here and changed the title to reflect that. Spoke to @junlincc and it now does address #11299

**Pull request merged**